src/v3/src/components/Widget/index.tsx

```diff
@@ -319,6 +320,12 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
 
   // track previous idxTransaction
   useEffect(() => {
+    // only abort if the steps don't contain webauthn autofill
+    // in order to have the browser suggestion popup work on clicking the username input field
+    if (!idxTransaction?.availableSteps?.find(({ name }) => name === IDX_STEP.CHALLENGE_WEBAUTHN_AUTOFILLUI_AUTHENTICATOR)) {
+      abortController?.abort();
+    }
```

Comment:
The reason for this is because the transaction for the webauthn autofill gets kicked off on load of the identify view. If the user decides not to use the passkey, but clicks on the next button and the next step is a webauthn authentication, we get an error saying that there is already a transaction pending.
We had to expose this abort controller in order to abort the pending transaction.